### DISCUSSION TITLE
Is there way to xonsh to remember last working directory before closing terminal?

### DISCUSSION THREAD

**Discussion by AnzenKodo** [xonsh/xonsh#4946] [Q&A] [answered]
I don't want to `cd` last working directory, every time I open terminal. Is there way to xonsh get right back to last working directory. Just like fish shell:
```fish
set -q fish_most_recent_dir && [ -d "$fish_most_recent_dir" ] && cd "$fish_most_recent_dir"

function save_dir --on-variable PWD
    set -U fish_most_recent_dir $PWD
end
```

**Comment by anki-code** [comment, marked as answer]
Just install [xontrib-back2dir](https://github.com/anki-code/xontrib-back2dir) and take a look into [other xontribs](https://github.com/topics/xontrib).
